docs/changelog.md

```diff
@@ -0,0 +1,68 @@
+## Changelog
+
+### v0.7.0(31/12/2020)
+
+- Add more evaluation metrics
+- Fix bugs
+
+#### New Features
+
+- Remove installation of MMCV from requirements. (#90)
+- Add more evaluation metrics: precision, recall and F-1 score. (#93)
+- Allow overriding configs through input argument when testing and inferencing. (#91 & #96)
+
+#### Bug Fixes
+
+- Add `CLASSES` in dataset wrappers. (#66)
+- Fix `round_up` parameter in evaluation during training. (#69)
```

Comment:
Commit message "Add missing CLASSES argument to dataset wrappers" and "Fix slurm evaluation error during training" might be better